src/evt_drivers/evt_pins.c

```diff
@@ -1,20 +1,31 @@
 #include "evt_pins.h"
 
 #include "chip.h"
-void Evt_GPIO_Init(void) {
-    // LEDs
-    Chip_GPIO_SetPinDIROutput(LPC_GPIO, LED0);
-    Chip_GPIO_SetPinDIROutput(LPC_GPIO, LED1);
-    Chip_GPIO_SetPinDIROutput(LPC_GPIO, LED2);
-    Chip_IOCON_PinMuxSet(LPC_IOCON, IOCON_LED2, IOCON_FUNC1);	
-    Chip_GPIO_WriteDirBit(LPC_GPIO, LED0, true);
-    Chip_GPIO_WriteDirBit(LPC_GPIO, LED1, true);
-    Chip_GPIO_WriteDirBit(LPC_GPIO, LED2, true);
 
+// CTR Pin
+#define EVT_CTR_SWTCH 1, 4 // pin 40 on lpc24, used as input 
+#define EVT_IOCON_CTR_SWTCH IOCON_PIO1_4
+#define EVT_IOCON_CTR_SWTCH_FUNC IOCON_FUNC0
+
+// Headroom Pin
+#define EVT_HEADROOM 3, 2
+#define EVT_IOCON_HEADROOM IOCON_PIO3_2
+#define EVT_IOCON_HEADROOM_FUNC IOCON_FUNC1
```

Comment:
@skoppula @ericponce are you guys intending to use this as a non-GPIO pin? The LPC manual says that FUNC1 is for the DCD functionality (which is what you were setting it as before), but in master, this is called as if it was a GPIO with SetPinState etc

![screen shot 2017-04-17 at 6 39 00 pm](https://cloud.githubusercontent.com/assets/5010391/25107399/24f1dbe6-239d-11e7-96f4-cdd3ba4af77f.png)